tests/test__default.py

```diff
@@ -85,7 +85,6 @@ def test__load_credentials_from_file_authorized_user_bad_format(tmpdir):
         _default._load_credentials_from_file(str(filename))
 
     assert excinfo.match(r'Failed to load authorized user')
-    assert excinfo.match(r'missing fields')
```

Comment:
@jonparrott Before merging, are you OK with dropping this? It disappears because the "raised from" is no longer **inside** the exception caught by this test. I suppose if we wanted to be fancy we could check in `stderr` for the traceback, which would have "missing fields" in it.

Reply:
As long as the user still sees a helpful message, it's fine.

Reply:
Yeah that will be **less** true on Python 2 in cases where I changed

```python
raise FooExc(caught_exc)
```

to

```python
six.raise_from(FooExc(), caught_exc)
```

so we probably shouldn't merge this "as-is".

Reply:
Yeah, in some of these cases I was okay with shadowing the original exception, but changing things up a bit to include the original exception in the new exception's message in some way while also using raise_from would be fine.